doc/tutorials/pixel_data/compressing.rst

```diff
@@ -1,230 +1,425 @@
+in (unsigned) pixel data values. A `jls_error` of ``3`` therefore means that all
+pixels will be within 3 intensity units of the original.
 
-*JPEG-LS Near-lossless* is currently the only transfer syntax that supports
-lossy compression, which works similarly to lossless but requires
-passing one or more keyword parameters to control the image quality. In the
-case of *JPEG-LS Near-lossless* this means the `jls_error` parameter:
+In our second lossy JPEG-LS example we'll use a dataset with 16-bit signed integers,
+which is where the complication starts. The NEAR parameter is defined in terms of
+unsigned integers, so when used with signed values there can potentially be
+compression errors of up to the maximum bit-depth of the pixel data. To avoid this,
+the range of pixel values must be in the `closed interval
+<https://en.wikipedia.org/wiki/Interval_(mathematics)>`_::
+
+    [-2**(ds.BitsStored - 1) + jls_error, 2**(ds.BitsStored - 1) - 1 - jls_error]
+
+For example, with a *Bits Stored* of ``8`` and ``jls_error=3`` the pixels must be in the
+range [-125, 124].
 
 .. code-block:: python
 
     >>> from pydicom import examples
     >>> from pydicom.uid import JPEGLSNearLossless
     >>> ds = examples.ct
+    >>> ds.PixelRepresentation
+    1
+    >>> ds.BitsStored
+    16
+    >>> arr = ds.pixel_array
+    >>> arr.min(), arr.max()
+    (128, 2191)
     >>> ds.compress(JPEGLSNearLossless, jls_error=3)
 
-This will compress the existing *Pixel Data* with an absolute error of no more
-than 3 pixel intensity units.
+In this example the pixel values are within the allowed range so we don't
+need to do anything further. If that weren't the case you'd have to rescale
+the values or use a different compression method such as JPEG 2000 (recommended).
+
+More information on performing compression with JPEG-LS can be found in
+the :doc:`JPEG-LS encoding guide</guides/encoding/jpeg_ls>`.
+
+
+Compressing with JPEG 2000
+--------------------------
+
+The JPEG 2000 compression scheme is based on `ISO/IEC 15444-1
+<https://www.iso.org/standard/78321.html>`_/`ITU T.800
+<https://www.itu.int/rec/T-REC-T.800-201511-S/en>`_. The format is fairly well supported
+by third-party applications and it can compress images with a wide variety of
+properties, making it a good choice for compressing datasets.
+
+Two transfer syntaxes are available that use JPEG 2000 compression; *JPEG 2000 Lossless*
+and *JPEG 2000*. While the DICOM Standard allows *JPEG 2000* to be either lossy or
+lossless, when used for compression in *pydicom* it's always treated as being lossy in
+order to simplify its usage.
+
+**Lossless compression**
+
+As with RLE and JPEG-LS, performing lossless compression is straight-forward::
+
+    >>> from pydicom import examples
+    >>> from pydicom.uid import JPEG2000Lossless
+    >>> ds = examples.ct
+    >>> ds.compress(JPEG2000Lossless)
+
+For RGB pixel data, JPEG 2000 can perform multiple component transformation
```

Comment:
multi-component transformation?

Reply:
It's called multiple component transformation in the standard

Reply:
Ah, ok - thanks!